packages/react-sdk/docusaurus/docs/React/03-ui-components/01-overview.mdx

```diff
@@ -7,57 +7,71 @@ description: Overview of the UI components
 TODO create pages for the components
 :::
 
-:::warning
-Here is a list of things still to address on this page:
-
-- Show an example of how to use one of the pre-built components
-- Add more CSS theming examples
-
-(Whenever you add one of the sections please remove the point from the list.)
-:::
-
 ## Introduction
 
 There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in components ready for use.
 In addition, it allows for styling and custom theming options with custom CSS.
 
-For custom use cases it is also possible to create your own components to fully tailor the SDK to your needs. Please note that we suggest not to override a few, core components as those contain complex, low-level logic related to performance, the list of these components can be found in the TODO.
+For custom use cases it is also possible to create your own components to fully tailor the SDK to your needs. Please note that we suggest not to override a few, core components as those contain complex, low-level logic related to performance, the list of these components can be found in the [Core Components page](../../basics/core-components/).
 
 This page guides you through all the options you have with increasing levels of customization.
 
 ## Using pre-built components
 
-The SDK comes with a lot of pre-built components for the different elements of the UI.
+You can create a fully functioning video calling application with just a few lines of code using pre-built components:
+
+```tsx
+import '@stream-io/video-react-sdk/dist/css/styles.css';
+import {
+  CallControls,
+  StreamCall,
+  StreamTheme,
+  StreamVideo,
+  SpeakerLayout,
+} from '@stream-io/video-react-sdk';
+
+const MyApp = () => {
+  // Assuming you have the 'client' instance created
+  return (
+    <StreamVideo client={client}>
+      <StreamTheme>
+        <StreamCall
+          callType="default"
+          callId="my-first-call"
+          data={{ create: true }}
+          autoJoin
+        >
+          <SpeakerLayout />
+          <CallControls />
+        </StreamCall>
+      </StreamTheme>
+    </StreamVideo>
+  );
+};
+```
 
 ## Theming with CSS
 
 If you only want to customize the styling of the available components, there is a wide range of CSS variables that you can override.
 With that, you can achieve a significant amount of theming in your video product.
 
-The way this works is that there are various reserved variable names inside of our [styling package](TODO: link to styling package).
 To change them you can simply override them with your custom values.
```

Comment:
We may want to mention, that the override has to take place within the scope of `.str-video` class.

Reply:
👍 Updated the docs: https://github.com/GetStream/stream-video-js/pull/546/commits/a27bdac57c907c6fd756a15778a507d03d0174c9